provider/equinix/environ.go

```diff
@@ -403,9 +397,25 @@ while true; do
     exit 0
 done
 EOF`,
-		"(crontab -l 2>/dev/null; echo '@reboot bash /root/juju-fixups.sh') | crontab -",
```

Comment:
This crontab entry should not have been removed. The persistence that was added for iptables does not persist IP routes.  Perhaps we should run just the `ip routes` section of `juju-fixups.sh` in cron (or persist this in ip-up.d files).